### DISCUSSION TITLE
terraform module 실습코드가 이상해요

### DISCUSSION THREAD

**Discussion by yeoec82** [tedilabs/fastcampus-devops#44] [General]
아래같은 에러가 나서 봤더니..
Could not download module "vpc" (main.tf:5) source code from
│ "git::https://github.com/tedilabs/terraform-aws-network?ref=v0.24.0": error downloading
│ 'https://github.com/tedilabs/terraform-aws-network?ref=v0.24.0': git must be available and on the
│ PATH.

테라폼의 vpc 모듈에 가도
```
module "network" {
  source  = "tedilabs/network/aws"
  version = "0.24.0"
  # insert the 1 required variable here
} 
```

이런식으로 보이더라고여 
강의자료의의 링크로 접속이 되고 있지 않아요 해결해 주세요 ㅠ

**Comment by posquit0** [comment]
안녕하세요~ 강의 영상 번호와 해당 소스코드 위치를 알려주시면 문제 해결에 도움을 드릴 수 있을 것 같아요.

우선 주신 정보를 토대로는 `source = "tedilabs/network/aws"` 파트가 잘못된 것 같습니다.
본 실습 레포지토리 내 모든 테라폼 코드는 `source  = "tedilabs/network/aws//modules/vpc"` 와 같이 source가 구성되어 있습니다.

혹시나 영상에서 보내주신 코드와 같이 적혀있는 부분이 있다면 정확한 위치 공유주시면 살펴보겠습니다..!!

**Reply by yeoec82** [reply]
ㅎㅎㅎ 제가 똥멍청이 였네요;; 우분투를 새로 깔면서 git이 설치 안되어 있어서.. github에 접근을 못했나봐여
git 설치하고 잘 됩니다. 감사합니다. 즐거운 명정 보내세요 강사님
